hw6/mergeSort/mergeSort.c

```diff
@@ -0,0 +1,176 @@
+﻿#define _CRT_SECURE_NO_WARNINGS
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include "List.h"
+
+void moveElements(List* listFrom, List* listTo, int counter)
+{
+    for (int i = 0; i < counter; ++i)
+    {
+        addElement(listTo, getHeadName(listFrom), getHeadPhone(listFrom));
+        deleteHeadElement(listFrom);
+    }
+}
+
+int compareHeads(List* firstList, List* secondList, int key)
+{
+    if (key == 1)
+    {
+        return (strcmp(getHeadName(firstList), getHeadName(secondList)) < 0);
```

Comment:
```suggestion
        return strcmp(getHeadName(firstList), getHeadName(secondList)) < 0;
```
Только это булевое значение, а мы обещаем вернуть int. Просто `return strcmp(getHeadName(firstList), getHeadName(secondList));` было бы лучше